operational_analysis/types/asset.py

```diff
@@ -11,7 +10,12 @@ class AssetData(object):
     """
     This class wraps around a GeoPandas dataframe that contains
     metadata about the plant assets. It provides some useful functions
-    to work with this data (e.g., calculating nearest neighbors, etc.)
+    to work with this data (e.g., calculating nearest neighbors, etc.). 
+    
+    This class requires the geopandas package, which can be installed using:
```

Comment:
I would recommend making this a "Note" like in the sphinx example so that when someone reads the docstrings it is further highlighted (example below).
![image](https://user-images.githubusercontent.com/13874373/93636794-c0f48780-f9b1-11ea-8c93-26e5c071fffd.png)

Reply:
Good suggestion, it looks great in the documentation.